source/module_hamilt_general/module_surchem/corrected_energy.cpp

```diff
@@ -1,23 +1,23 @@
 #include "surchem.h"
 
-double surchem::cal_Ael(const UnitCell &cell, ModulePW::PW_Basis *rho_basis)
+double surchem::cal_Ael(const UnitCell &cell, const int& nrxx, const int& nxyz)
```

Comment:
Without comments, the meaning of this two parameters `nrxx` and `nxyz` will gradually become meaningless.

Reply:
OK, I have added the commnets.